src/test/java/org/broadinstitute/dsde/consent/ontology/datause/DataUseMatcherV3Test.java

```diff
@@ -0,0 +1,309 @@
+    DataUseV3 purpose = new DataUseBuilderV3().setGeneralUse(true).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testHMB_negative_case_2() {
+    DataUseV3 purpose = new DataUseBuilderV3().setHmbResearch(true).build();
+    DataUseV3 dataset = new DataUseBuilderV3().setDiseaseRestrictions(Collections.singletonList(cancerNode)).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testHMB_negative_case_3() {
+    DataUseV3 purpose = new DataUseBuilderV3().setHmbResearch(true).build();
+    DataUseV3 dataset = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testPOA_positive_case_1() {
+    DataUseV3 dataset = new DataUseBuilderV3().setGeneralUse(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    assertApprove(purpose, dataset);
+  }
+
+  @Test
+  public void testPOA_positive_case_2() {
+    DataUseV3 dataset = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    assertApprove(purpose, dataset);
+  }
+
+  @Test
+  public void testPOA_negative_case_1() {
+    DataUseV3 dataset = new DataUseBuilderV3().setHmbResearch(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testPOA_negative_case_2() {
+    DataUseV3 dataset = new DataUseBuilderV3()
+        .setDiseaseRestrictions(Collections.singletonList(cancerNode))
+        .setMethodsResearch(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testMDS_positive_case_1() {
+    DataUseV3 dataset = new DataUseBuilderV3().setGeneralUse(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setMethodsResearch(true).build();
+    assertApprove(purpose, dataset);
+  }
+
+  @Test
+  public void testMDS_positive_case_2() {
+    DataUseV3 dataset = new DataUseBuilderV3().setPopulationOriginsAncestry(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setMethodsResearch(true).build();
+    assertApprove(purpose, dataset);
+  }
+
+  @Test
+  public void testMDS_negative_case_1() {
+    DataUseV3 dataset = new DataUseBuilderV3().setMethodsResearch(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3()
+        .setDiseaseRestrictions(Collections.singletonList(cancerNode))
+        .setMethodsResearch(true).build();
+    assertDeny(purpose, dataset);
+  }
+
+  @Test
+  public void testMDS_negative_case_2() {
+    DataUseV3 dataset = new DataUseBuilderV3().setHmbResearch(true).build();
+    DataUseV3 purpose = new DataUseBuilderV3().setMethodsResearch(true).build();
+    assertDeny(purpose, dataset);
+  }
```

Comment:
I'm confused on these negative cases. For a MDS DAR, the docs say:
> Approve if the dataset's Primary DUO terms are GRU, DS-, HMB, POA

Case 1 should be a MR purpose against a DS dataset and, if the doc is correct, should be approved.
Case 2 should also be true if the doc is correct.

Reply:
I made a mistake here, I accidentally used the POA cases for MDS. I'll correct that now!